posix.mak

```diff
@@ -522,8 +522,8 @@ checkwhitespace: $(LIB) $(TOOLS_DIR)/checkwhitespace.d
 #############################
 
 ../dscanner:
-	git clone https://github.com/Hackerpilot/Dscanner ../dscanner
-	git -C ../dscanner checkout tags/v0.4.0
+	git clone https://github.com/dlang-community/Dscanner ../dscanner
+	git -C ../dscanner checkout phobos
```

Comment:
Using a feature branch may interfere with editor/IDE Dscanner integrations, is this temporary?